### ISSUE TITLE
Point cloud

### ISSUE THREAD

**Issue by HelliceSaouli** [andyzeng/tsdf-fusion#9]
Hello
I 'm just wondering is there a way to go back from Point cload to a TSDF volume ?

**Comment by chuyaqifei** [comment]
Hi~ I just read a paper about this. Before converting a pointcloud to a TSDF volume, the pose at which the point is observed should be saved. Then the ray casting method can be used to recover the occupancy of each volume.  The paper is called "Coxgraph ...." published in  IROS 2021.

**Comment by mpottinger** [comment]
> Hi~ I just read a paper about this. Before converting a pointcloud to a TSDF volume, the pose at which the point is observed should be saved. Then the ray casting method can be used to recover the occupancy of each volume.  The paper is called "Coxgraph ...." published in  IROS 2021.

Yes I have learned something similar from the vdbfusion library. https://github.com/PRBonn/vdbfusion

This repository here was the first that helped me understand how tsdf fusion works, but the disadvantage is that it operates on a fixed size dense voxel grid. Vdbfusion showed me it is possible to do tsdf fusion in a sparse grid with a pointoud and camera pose as input.
